packages/api/src/enrichments/searcher-constituents-wikidata.integration.test.ts

```diff
@@ -0,0 +1,68 @@
+import {WikidataConstituentSearcher} from './searcher-constituents-wikidata';
+import {beforeEach, describe, expect, it} from '@jest/globals';
+
+let constituentSearcher: WikidataConstituentSearcher;
+
+beforeEach(() => {
+  constituentSearcher = new WikidataConstituentSearcher({
+    endpointUrl: 'https://query.wikidata.org/sparql',
+  });
+});
+
+describe('search', () => {
+  it('finds constituents that match the query in the specified locale', async () => {
+    const result = await constituentSearcher.search({
+      query: 'rembrandt',
+      locale: 'nl',
+      limit: 3,
+    });
+
+    expect(result).toStrictEqual({
+      things: [
```

Comment:
I didn't expect the word `things` here. I understand it comes from the type `Thing`. Most APIs use the term `data` for the data response, but that is even more generic than a `thing`. `constituents` is also an option, but maybe that is not generic enough. So I'm not sure about the best option. you can ignore this comment.

Reply:
Thanks! Yes, `things` could sound a bit strange. It's a term from the base data model that the API uses, https://schema.org/Thing. The term `constituents` is an option, but I looked for a generic name that also allows other types of 'things', such as locations.